Content.Server/Chemistry/ReagentEffects/PlantMetabolism/PlantAdjustAttribute.cs

```diff
@@ -15,6 +15,18 @@ public abstract partial class PlantAdjustAttribute : ReagentEffect
         [DataField]
         public float Prob { get; protected set; } = 1; // = (80);
 
+        /// <summary>
+        /// Localisation key for the name of the adjusted attribute. Used for guidebook description.
+        /// </summary>
+        [DataField]
+        public virtual string GuidebookAttributeName { get; set; } = "plant-attribute-missing";
```

Comment:
Make this `abstract`, not `virtual`. It should be required to specify this, without a default value fallback. Otherwise people *will* forget and reviewers *will* miss it.

Reply:
Aye, haven't gone into OOP in at least quite a while and ended up going a few wrong ways trying to make this work and this was what I landed on.
Anyway, yeah, `abstract` works too and you've made a fine point why to use that, so made that change.